region error: Greece

While working on this pull request, https://github.com/openENTRANCE/openentrance/pull/154, I got an unrelated error in the automated tests:

```
=================================== FAILURES ===================================
_______________________________ test_iso_mapping _______________________________

    def test_iso_mapping():
        # check that iso-mapping dictionary is not empty and has specific elements
        for name in ["GR", "GRC", "EL"]:
>           assert iso_mapping[name] == "Greece"
E           KeyError: 'GR'

tests/test_core.py:7: KeyError
_____________________________ test_nuts_hierarchy ______________________________

    def test_nuts_hierarchy():
        # check that nuts-hierarchy is not empty and has specific elements
>       assert nuts_hierarchy["Belgium"]["BE2"]["BE24"] == ["BE241", "BE242"]
E       KeyError: 'Belgium'

tests/test_core.py:12: KeyError
=========================== short test summary info ============================
FAILED tests/test_core.py::test_iso_mapping - KeyError: 'GR'
FAILED tests/test_core.py::test_nuts_hierarchy - KeyError: 'Belgium'
======================== 2 failed, 18 passed in 55.[69](https://github.com/openENTRANCE/openentrance/runs/5183941245?check_suite_focus=true#step:5:69)s =========================
Error: Process completed with exit code 1.
```

--- a/openentrance/__init__.py
+++ b/openentrance/__init__.py
@@ -4,11 +4,11 @@ from nomenclature import CodeList
 
 
 # path to nomenclature definitions
-DEF_PATH = Path(__file__).parent.parent / "definitions"
+DEF_PATH = Path(__file__).parent.parent / "definitions" / "region"
 
 
 countries = CodeList.from_directory(
-    "region", path=DEF_PATH / "region", file="countries"
+    "region", path=DEF_PATH, file="countries.yaml"
 )
 """CodeList of countries"""
 
@@ -34,7 +34,7 @@ def _create_nuts_hierarchy():
     """Parse nuts3.yaml and create hierarchical dictionary"""
 
     hierarchy = dict()
-    nuts3 = CodeList.from_directory("region", path=DEF_PATH / "region", file="nuts3")
+    nuts3 = CodeList.from_directory("region", path=DEF_PATH, file="nuts3.yaml")
 
     for n3, mapping in nuts3.items():
         country, n1, n2 = [mapping.get(i) for i in ["country", "nuts1", "nuts2"]]

